toqito/state_props/tests/test_common_quantum_overlap.py

```diff
@@ -0,0 +1,66 @@
+"""Unit tests for the common_quantum_overlap function."""
+
+import numpy as np
+import pytest
+
+from toqito.state_props import common_quantum_overlap
+from toqito.states import bell
+
+
+@pytest.mark.parametrize(
+    "states, expected_overlap, description",
+    [
+        (
+            # Bell states are perfectly antidistinguishable (ω_Q = 0) as per arXiv:2401.17980v2.
+            [bell(0), bell(1), bell(2), bell(3)],
+            0,
+            "Bell states are perfectly antidistinguishable (ω_Q = 0) as per arXiv:2401.17980v2.",
```

Comment:
Again, I would remove the description field here and just use the comment to describe the test.